.github/workflows/build-standalone.yml

```diff
@@ -40,6 +40,60 @@ jobs:
           security import codesign_certificate.p12 -k build.keychain -P "${{ secrets.APPLE_DEVELOPER_ID_CERTIFICATE_PASSWORD }}" -T /usr/bin/codesign
           security import installer_certificate.p12 -k build.keychain -P "${{ secrets.APPLE_DEVELOPER_ID_CERTIFICATE_PASSWORD }}" -T /usr/bin/pkgbuild
           security set-key-partition-list -S apple-tool:,apple: -s -k "$KEYCHAIN_PASSWORD" build.keychain
+      - name: Cache FIPS-compliant Node.js
+        id: cache-fips-node
+        uses: actions/cache@v4
+        with:
+          path: ~/fips-node
+          key: fips-node-${{ hashFiles('.nvmrc') }}-${{ runner.os }}-openssl3.0.9-v1
+      - name: Build FIPS-compliant Node.js
+        if: steps.cache-fips-node.outputs.cache-hit != 'true'
+        run: |
+          # Install build dependencies
+          brew install python@3.11
+          export PATH="/opt/homebrew/bin:$PATH"
+
+          # Create build directory
+          mkdir -p ~/fips-node-build
+          cd ~/fips-node-build
+
+          # Build OpenSSL 3.0.9 with FIPS support
+          curl -L https://www.openssl.org/source/openssl-3.0.9.tar.gz | tar -xz
+          cd openssl-3.0.9
+          ./Configure darwin64-arm64-cc --prefix=$HOME/fips-node/openssl enable-fips
+          make -j$(sysctl -n hw.ncpu)
```

Comment:
Use `getconf _NPROCESSORS_ONLN` instead of `sysctl -n hw.ncpu`, it's more portable.